compiler/rustc_mir_build/src/build/mod.rs

```diff
@@ -967,13 +967,60 @@ fn parse_float_into_constval<'tcx>(
     parse_float_into_scalar(num, float_ty, neg).map(ConstValue::Scalar)
 }
 
+// #[cfg(not(bootstrap))]
+// fn parse_f16(num: &str) -> Option<f16> {
+//     num.parse().ok()
+// }
+
+// FIXME: bootstrap `f16` parsing via `f32`
+// #[cfg(bootstrap)]
+fn parse_f16(num: &str) -> Option<f32> {
+    num.parse().ok()
+}
+
+// #[cfg(not(bootstrap))]
+// fn parse_f128(num: &str) -> Option<f128> {
+//     num.parse().ok()
+// }
+
+// FIXME: bootstrap `f16` parsing via `f32`
```

Comment:
```suggestion
// FIXME: bootstrap `f128` parsing via `f64`
```

Reply:
Also please don't remove this until both cg_clif and cg_gcc support the new types. Otherwise they can't bootstrap rustc anymore.